pallets/stake/src/tests.rs

```diff
@@ -910,11 +910,11 @@ fn payouts_follow_nomination_changes() {
 		);
 		assert_ok!(Stake::leave_nominators(Origin::signed(6)));
 		roll_to(21);
-		// keep paying 6 (note: inflation is in terms of total issuance so that's why 1 is 21)
```

Comment:
The reason these payout values changed is because before we were computing issuance at the time of payment. Remember that computing issuance uses the current total issuance as the base so when we compute it later, it will be larger.

Now we are computing issuance at the beginning of the round so it is less because the base issuance is less at this time.